models/artifact_helpers.go

```diff
@@ -137,6 +155,10 @@ func CreateArtifact(q *reform.Querier, params CreateArtifactParams) (*Artifact,
 		return nil, errors.WithStack(err)
 	}
 
+	if err := checkUniqueArtifactName(q, params.Name); err != nil {
+		return nil, err
+	}
```

Comment:
Added constraint for artifact name uniqueness. It should be also DB constraint, but we already released schema without it. Probably we can cleanup artifacts table in migration and apply constraint until backups are in tech preview state.

Reply:
Why artifact name should be unique? don't we add datetime to artifact name?

Reply:
That restriction was suggested by @rnovikovP to simplify MVP. For now we use name exactly how it was specified by user, but it seems that we need really unique names for backups, because backup tooling will use name to determine what to restore. So we should be more or less sure that there is no second backup with same name.

P.s. backup name is the `folder` name in S3, so two backups with the same name will just put their files in the same directory, but it can cause problems.